lib/Widgets/DynamicNotebook.vala

```diff
@@ -668,6 +695,45 @@ namespace Granite.Widgets {
         // Use temporary field to avoid breaking API this can be dropped while preparing for 0.4
         string _add_button_tooltip;
 
+        public string add_button_tooltip_markup {
+            owned get { return add_button.get_tooltip_markup (); }
+            set { add_button.tooltip_markup = value; }
+        }
+
+        /**
```

Comment:
I think these comments need to be right above the public properties to show in the docs